pages/common/gitlab-runner.md

```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> A CLI tool for managing your GitLab runners.
```

Comment:
Perhaps this will read better?
```suggestion
> CLI tool for managing GitLab runners.
```

Reply:
See https://github.com/tldr-pages/tldr/pull/4758#discussion_r508295350.
Is there a consensus for this kind of things?